### PR TITLE
[Fix]modify dict keys of sot tracking outputs

### PR DESCRIPTION
**Pull request by JingweiZhang12**
replace "score" and "bbox" to "track_results" in dict keys of sot tracking outputs

### CODE REVIEW

**Comment by GT9505** [comment]
please fix the lint error firstly.

File: mmtrack/models/sot/siamrpn.py [hunk 1]
```diff
@@ -272,10 +273,11 @@ def simple_test(self, img, img_metas, gt_bboxes, **kwargs):
...
         results = dict()
         if best_score is None:
-            results['score'] = best_score
+            results['track_results'] = np.concatenate(
+                (np.array([best_score]), bbox_pred.cpu().numpy()))
         else:
-            results['score'] = best_score.cpu().numpy()
-        results['bbox'] = bbox_pred.cpu().numpy()
+            results['track_results'] = np.concatenate(
+                (best_score.cpu().numpy()[None], bbox_pred.cpu().numpy()))
```
Comment (GT9505) [inline mmtrack/models/sot/siamrpn.py:280]: We usually put the `bbox` in the front of `score`.

File: mmtrack/datasets/lasot_dataset.py [hunk 2]
```diff
@@ -75,7 +75,8 @@ def evaluate(self, results, metric=['track'], logger=None):
             inds.append(len(self.data_infos))
 
             track_results = [
-                results['bbox'][inds[i]:inds[i + 1]] for i in range(num_vids)
+                results['track_results'][inds[i]:inds[i + 1]][:4]
```
Comment (GT9505) [inline mmtrack/datasets/lasot_dataset.py:78]: change `track_results` to `track_bboxes`

File: mmtrack/datasets/lasot_dataset.py [hunk 3]
```diff
@@ -75,7 +75,8 @@ def evaluate(self, results, metric=['track'], logger=None):
             inds.append(len(self.data_infos))
 
             track_results = [
```
Comment (GT9505) [inline mmtrack/datasets/lasot_dataset.py:77]: Change `track_results` to `track_bboxes`.
Please also fix the unittest errors.

Comment (GT9505) [reply mmtrack/datasets/lasot_dataset.py:77]: we should change the code to :
`track_bboxes = [`
`    results['bbox'][inds[i]:inds[i + 1]][:4]`
`    for i in range(num_vids)`
`]`

File: tests/test_data/test_dataset.py [hunk 4]
```diff
@@ -452,7 +452,7 @@ def test_lasot_evaluation():
         x1, y1, x2, y2 = result.split(',')
         track_results.append(np.array([int(x1), int(y1), int(x2), int(y2)]))
```
Comment (GT9505) [inline tests/test_data/test_dataset.py:453]: The `track_results` need append a score to be consistent with the above code.

File: mmtrack/datasets/lasot_dataset.py (hunk 3 above)
Comment (JingweiZhang12) [reply mmtrack/datasets/lasot_dataset.py:77]: results['track_results'] is the list[list[nparray]] format

File: mmtrack/models/sot/siamrpn.py [hunk 5]
```diff
@@ -262,7 +263,7 @@ def simple_test(self, img, img_metas, gt_bboxes, **kwargs):
             self.memo.bbox = bbox_xyxy_to_cxcywh(gt_bboxes)
             self.memo.z_feat, self.memo.avg_channel = self.init(
                 img, self.memo.bbox)
-            best_score = None
+            best_score = -1
```
Comment (GT9505) [inline mmtrack/models/sot/siamrpn.py:266]: `-1.` We need be careful about the type of variables

File: tools/test.py [hunk 6]
```diff
@@ -173,7 +173,10 @@ def main():
         if args.eval:
             eval_kwargs = cfg.get('evaluation', {}).copy()
             # hard-code way to remove EvalHook args
-            for key in ['interval', 'tmpdir', 'start', 'gpu_collect']:
+            for key in [
+                    'interval', 'tmpdir', 'start', 'gpu_collect', 'save_best',
+                    'rule', 'by_epoch'
+            ]:
```
Comment (GT9505) [inline tools/test.py:179]: Too ugly here. we can modify as following:
`eval_hook_args = [ 'interval', 'tmpdir', 'start', 'gpu_collect', 'save_best' , 'rule', 'by_epoch']`
`for key in eval_hook_args:`

File: tests/test_data/test_dataset.py [hunk 7]
```diff
@@ -450,9 +450,9 @@ def test_lasot_evaluation():
     track_results = []
     for result in results:
         x1, y1, x2, y2 = result.split(',')
-        track_results.append(np.array([int(x1), int(y1), int(x2), int(y2)]))
+        track_results.append(np.array([int(x1), int(y1), int(x2), int(y2), 0]))
```
Comment (GT9505) [inline tests/test_data/test_dataset.py:453]: change all `int` to `float`

**Review by GT9505 — APPROVED** [review]

**Pull request merged**